custom_components/opnsense/coordinator.py

```
@@ -119,144 +106,159 @@
+                    "inpkts",
+                    "outpkts",
+                    # "inpktspass",
+                    # "outpktspass",
+                    # "inpktsblock",
+                    # "outpktsblock",
+                ]:
+                    try:
+                        current_parent_value: float = interface[prop_name]
+                        previous_parent_value: float = previous_interface[prop_name]
+                        change: float = abs(
+                            current_parent_value - previous_parent_value
```

Ruff

Trailing comma missing

Add trailing comma

(COM812)